reactions/include/Copter/HALO.h

```diff
@@ -5,12 +5,11 @@
 
 // limits on mass exponents
```

Comment:
Suggestion: improve comment to explain how these are chosen.

Reply:
added comments on how and why these values are chosen